common/src/main/java/bisq/common/util/Tuple5.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * This file is part of Bisq.
+ *
+ * Bisq is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or (at
+ * your option) any later version.
+ *
+ * Bisq is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package bisq.common.util;
+
+import java.util.Objects;
+
+public class Tuple5<A, B, C, D, E> {
+    final public A first;
+    final public B second;
+    final public C third;
+    final public D forth;
```

Comment:
fourth

Reply:
ah damn English 😉

Reply:
or let's say damn copy & paste. I'll fix it in the Tuple4 as well